src/ucs/debug/debug.c

```diff
@@ -17,6 +17,7 @@
 #include <ucs/sys/string.h>
 #include <ucs/sys/math.h>
 #include <ucs/sys/sys.h>
+#include <ucs/type/spinlock.h>
```

Comment:
looks like an spinlock's implementation was missed (the file is not under git), isn't it?

Reply:
I used the orignal  spinlock implementation c303724fd7bc85ef9609f92c1b5cd575b6a1e1f4
I didn't implement spinlock in this PR.

Reply:
got it
I didn't know that we have `ucs_spinlock`

pls, nevermind comments below

Reply:
actually there is spinlock implementation where added some useful tips, like thread owner and lock count :)

Reply:
> actually there is spinlock implementation where added some useful tips, like thread owner and lock count :)

yes, I haven't checked it before leaving my comments
I thought @hiroyuki-sato are going to add this abstraction